mindsdb/integrations/handlers/vertex_handler/vertex_handler.py

```diff
@@ -0,0 +1,53 @@
+from mindsdb.integrations.libs.base import BaseMLEngine
+from mindsdb.integrations.handlers.vertex_handler.vertex_client import VertexClient
+import pandas as pd
+
+
+class VertexHandler(BaseMLEngine):
+    """Handler for the Vertex Google AI cloud API"""
+
+    name = "Vertex"
+
+    def create(self, target, df, args={}):
+        """Logs in to Vertex and deploy a pre-trained model to an endpoint.
+
+        If the endpoint already exists for the model, we do nothing.
+
+        If the endpoint does not exist, we create it and deploy the model to it.
+        The runtime for this is long, it took 15 minutes for a small model.
+        """
+        model_name = args["using"]["model_name"]
```

Comment:
Perhaps for clearer error messages: `assert 'using' in args, "This handler requires USING syntax"`